### ISSUE TITLE
Improve the readme

### ISSUE THREAD

**Issue by popooff** [nick92/flutter_mapbox#10]
Can you improve the readme and add more information? The first setup and how we can use it easily.

**Comment by nick92** [comment]
Yup, definitely on my TODO, will look at getting around to this by the end of next week

**Comment by nick92** [comment]
updated now
